Controllers/EmployeeManagement/EmployeesController.cs

```diff
@@ -10,30 +12,42 @@ namespace PSP_Komanda32_API.Controllers.EmployeeManagement
     [ApiExplorerSettings(GroupName = "Manage employees")]
     public class EmployeesController : ControllerBase
     {
-        readonly IRandomizer _randomizer;
+        //readonly IRandomizer _randomizer;
 
-        public EmployeesController(IRandomizer randomizer)
+        //public EmployeesController(IRandomizer randomizer)
+        //{
+        //    _randomizer = randomizer;
+        //}
+
+        private readonly PoSSContext _context;
+
+        public EmployeesController(PoSSContext context)
         {
-            _randomizer = randomizer;
+            _context = context;
         }
 
+        [BindProperty]
+        public List<Employee> Employee { get; set; }
```

Comment:
Probably we should have an employee service or repository that accesses this field directly

Reply:
Yeah will change up a bit how everything works on the controller once I get to working on it